src/CareTogether.Api/OData/LiveODataModelController.cs

```diff
@@ -362,17 +369,91 @@ private async Task<LiveModel> RenderLiveModelAsync()
             var organizationId = userDataAccessScope.OrganizationId;
             var anonymize = userDataAccessScope.Anonymize;
 
-            var result = await cache.GetOrAddAsync(
-                $"LiveODataModelController-RenderLiveModelAsync-{organizationId}-{(anonymize ? "ANON" : "PII")}",
-                async cacheEntry =>
-                {
-                    var result = await RenderLiveModelInternalAsync(organizationId, anonymize);
+            var isGlobal = userDataAccessScope.IsGlobal;
 
-                    cacheEntry.SlidingExpiration = TimeSpan.FromMinutes(1);
-                    cacheEntry.AbsoluteExpiration = DateTimeOffset.Now.AddMinutes(5);
+            var organizationIds = isGlobal
+                ? await accountsResource.GetValidOrganizationsAsync()
+                : [organizationId];
+
+            var results = await organizationIds
+                .Select(async organizationId =>
+                    await cache.GetOrAddAsync(
+                        $"LiveODataModelController-RenderLiveModelAsync-{organizationId}-{Guid.NewGuid()}-{(anonymize ? "ANON" : "PII")}",
+                        async cacheEntry =>
+                        {
+                            var result = await RenderLiveModelInternalAsync(
+                                organizationId,
+                                anonymize
+                            );
+
+                            cacheEntry.SlidingExpiration = TimeSpan.FromMinutes(1);
+                            cacheEntry.AbsoluteExpiration = DateTimeOffset.Now.AddMinutes(5);
+
+                            return result;
+                        }
+                    )
+                )
+                .WhenAll();
 
-                    return result;
-                }
+            var result = results.Aggregate(
+                new LiveModel(
+                    Enumerable.Empty<Location>(),
+                    Enumerable.Empty<LocationUserAccess>(),
+                    Enumerable.Empty<Family>(),
+                    Enumerable.Empty<Person>(),
+                    Enumerable.Empty<Community>(),
+                    Enumerable.Empty<Role>(),
+                    Enumerable.Empty<FamilyRoleApproval>(),
+                    Enumerable.Empty<IndividualRoleApproval>(),
+                    Enumerable.Empty<FamilyRoleRemovedIndividual>(),
+                    Enumerable.Empty<Referral>(),
+                    Enumerable.Empty<Arrangement>(),
+                    Enumerable.Empty<ArrangementType>(),
+                    Enumerable.Empty<ChildLocationRecord>(),
+                    Enumerable.Empty<FamilyFunctionAssignment>(),
+                    Enumerable.Empty<IndividualFunctionAssignment>(),
+                    Enumerable.Empty<CommunityMemberFamily>(),
+                    Enumerable.Empty<CommunityRoleAssignment>()
+                ),
+                (acc, model) =>
+                    acc with
```

Comment:
It's safer to write this code as `new LiveModel(...)` instead of `acc with { ... }` since the constructor syntax will force future developers to include all properties, e.g. if a new entity set is added to `LiveModel`.